coverage-experiment/src/main/java/nl/tudelft/serg/evosql/experiment/Runner.java

```diff
@@ -1,18 +1,135 @@
-        //     3b. Store fixture that GA outputs
-        //     3c. Construct test class with fixture for original query
-        //     3d. Mutate query
-        //     3e. Construct test class with fixture for mutated query
-        // 4. Execute all tests that should pass and store results
-        // 5. Execute all tests that shouldn't pass and store results
-        // 6. Output final results
+        int startIndex = Integer.valueOf(args[0]);
+        int stepSize = Integer.valueOf(args[1]);
+        BufferedReader reader_erpnext = new BufferedReader(new InputStreamReader(
+                Runner.class.getResourceAsStream("sql/erpnext_queries.sql")));
+        Stream<String> erpnext = reader_erpnext.lines();
+
+        BufferedReader reader_espocrm = new BufferedReader(new InputStreamReader(
+                Runner.class.getResourceAsStream("sql/espocrm_queries.sql")));
+        Stream<String> espocrm = reader_espocrm.lines();
+
+        BufferedReader reader_suitecrm = new BufferedReader(new InputStreamReader(
+                Runner.class.getResourceAsStream("sql/suitecrm_queries.sql")));
+        Stream<String> suitecrm = reader_suitecrm.lines();
+
+
+        QueryReader queryReader = new QueryReader();
+        List<String> allQueries = queryReader.readQueries(erpnext, espocrm, suitecrm);
+
+        // TODO: Make connection data for separate databases...
+        for (int i = startIndex; i < allQueries.size(); i += stepSize) {
+            runForQuery(
+                    allQueries.get(i),
+                    null,
+                    null,
+                    String.valueOf(i)
+            );
+        }
+
+        try {
+            reader_erpnext.close();
+            reader_espocrm.close();
+            reader_suitecrm.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
     }
+
+    /**
+     * End to end execution of experiment for a single query.
+     *
+     * @param query              query to execute
+     * @param connectionDataProd data of db connection for production database
+     * @param connectionDataTest data of db connection for test database
+     */
+    public static QueryExperimentResult runForQuery(String query,
+                                             ConnectionData connectionDataProd,
+                                             ConnectionData connectionDataTest,
+                                             String queryClassName) {
+        // TODO: Get file package
+        BrewExecutor brewExecutor = new BrewExecutor(connectionDataProd, connectionDataTest, "");
+        // TODO: Pass a good folder name which is useable
+        brewExecutor.executeBrew(query, Paths.get("../test"), queryClassName + "_original.java");
+
+        //String mutatedQuery = QueryMutator.mutateQuery(query);
+        brewExecutor.brewWithMutatedQuery(query, brewExecutor.getQueryResult(), Paths.get("../test"), queryClassName + "_mutated.java");
```

Comment:
We will need to create a new `Result` object with the same properties as the one generated through GA, but with a different production query set. The query given to `BrewExecutor` does not suffice as that is only used to run GA on.

Reply:
This method is completely incorrect since we will change our approach for executing the files, so I will fix this.